src/libcore/str/mod.rs

```diff
@@ -299,6 +299,20 @@ unsafe fn from_raw_parts_mut<'a>(p: *mut u8, len: usize) -> &'a mut str {
 ///
 /// assert_eq!("💖", sparkle_heart);
 /// ```
+///
+/// Incorrect bytes:
+///
+/// ```
```

Comment:
Actually it might? The `from_utf8_unchecked` won't fail, but printing it might? Not sure.